ansible_collections/serverscom/sc_api/plugins/modules/sc_l2_segments_info.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2020, Servers.com
+# GNU General Public License v3.0
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = """
+---
+module: sc_l2_segments_info
+version_added: "1.0.0"
+author: "George Shuklin (@amarao)"
+short_description: Information about existing L2 segments.
+description: >
+    Returns all L2 segments
+
+options:
+
+    endpoint:
+      type: str
+      default: https://api.servers.com/v1
+      description:
+        - Endpoint to use to connect to API.
```

Comment:
Too many 'to's for me. I'd suggest renaming this to "API endpoint"

Reply:
I'll address that in a separate MR, as it's repeated across all modules.